packages/edit-post/src/components/preferences-modal/style.scss

```diff
@@ -81,7 +81,7 @@
 		}
 	}
 
-	&__section {
+	&__section:not(:last-child) {
```

Comment:
This is the only one I don't love. Although I like that `:not` exists as a tool, I find that it has come back to bite me many times in the past due to the extra specificity it adds.

If there's an alternative we could use, that would be preferable. Otherwise, probably fine.

Reply:
No sweat to flip this around 👍